_posts/en/newsletters/2024-02-14-newsletter.md

```diff
@@ -0,0 +1,200 @@
+release candidates.*
+
+<!-- FIXME: harding to update Tuesday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+- [Bitcoin Core #28948][] adds support for [version 3 transaction
+  relay][topic v3 transaction relay] allowing any v3 transaction that
+  has no unconfirmed parent to enter the mempool according to the normal
+  transaction acceptance rules.  The v3 transaction can be [CPFP fee
+  bumped ][topic cpfp] but only if the child is 1,000 vbytes or less.
+  Each v3 parent may only have one unconfirmed child transaction in the
+  mempool and each child may only have one unconfirmed parent.  Either
+  the parent or child transaction can always be [replaced by fee][topic
+  rbf].  The rules only apply to Bitcoin Core's relay policy; at the
+  consensus layer, v3 transactions are validated the same as version 2
+  transactions defined in [BIP68][].  The new rules are intended to help
+  contract protocols such as LN ensure their precommitted transactions
+  can always be confirmed quickly with minimal extra fees needed to
+  escape [transaction pinning][topic transaction pinning].
+
+- [Core Lightning #6785][] makes [anchor-style][topic anchor outputs]
+  channels the default on Bitcoin.  Non-anchor channels are still used
+  for channels on Elements-compatible [sidechains][topic sidechains].
+
+- [Eclair #2818][] maximizes the number of inputs the Eclair wallet
+  believes it can safely spend by detecting some cases when an existing
+  unconfirmed transaction is very unlikely to become confirmed.  Eclair
+  uses Bitcoin Core's wallet to manage its UTXOs for onchain spending,
+  including for fee bumping transactions.  When a UTXO controlled by the
+  wallet is used as an input in a transaction, Bitcoin Core's wallet won't
+  automatically create other unrelated transactions using that input.
+  However, if that transaction becomes unconfirmable because a different
+  input in that transaction was double spent, Bitcoin Core's wallet will
+  automatically allow the UTXO to be spent in a different transaction
+  again.  Unfortunately, if a parent of the transaction is made
+  unconfirmable because a different version was confirmed, Bitcoin
+  Core's wallet will not currently automatically allow the UTXO to be
+  spent.  Eclair can independently detect a double spend of the parent
+  transaction and it will now tell Bitcoin Core's wallet to manually
+  unlock the UTXO so it can be spent again.
```

Comment:
```suggestion
  abandon the transaction so the UTXO can be spent again.
```

And perhaps link to https://developer.bitcoin.org/reference/rpc/abandontransaction.html